src/tqec/templates/orchestrator.py

```diff
@@ -332,19 +349,30 @@ def build_array(self, indices_map: tuple[int, ...]) -> numpy.ndarray:
             ret[y : y + tshapey, x : x + tshapex] = template.instanciate(
                 *plaquette_indices
             )
-        return ret
+            increments[y : y + tshapey, x : x + tshapex] = template.get_increments()
+        return ret, increments
```

Comment:
I do not get the point of returning the increments array. If we have the constraint that all the template within a orchestrator should have the same increments along each dimension, why do we need to compute and save this increments array?

Reply:
@nelimee, what do you think? I also felt like we were moving in circles with this one. We basically made the shape into the default increment.

Reply:
In the case of non-overlapping plaquettes, I agree that this seems like a reformulation of the original code. I am even questioning the lifting of the assumption right now. I do not have an answer to that for the moment, sorry.

Reply:
I think the underlying assumption we should lift is that we need to know the "shape" of the plaquette. The template can be totally unaware of the "shape" of the plaquette to be well defined. And more, the plaquettes can also be well defined with the local coordinate system and the qubits offsets without the shape property.

Reply:
For @afowler comment on the Google group, it seems like we misunderstood the requirements. We only need to be able to represent rectangles plaquette. For a given QEC, all the plaquettes will be the same (i.e., the same rectangle, the same square, ...).
This was already possible with the `shape` implementation, but as noted above it is beneficial to remove completely the `shape` information in favour of an increment between plaquettes.

So to summarise, we "simply" want to have a `(increment_x, increment_y)`, global to a given QEC, in order to place correctly the plaquette circuits.

I am missing something else here?

I am sorry for misunderstanding the root of issue #34.

Reply:
So if I understood correctly, I can remove again the `increments` information from the templates and move it to the the orchestrator?

Reply:
Seems like the `increments` data is only needed in the `generate_circuit` function. 

As a side note, even though this has not been done yet and the class hierarchy does not reflect that, the `TemplateOrchestrator` class has been thought as a `Template` with the idea that we could, in the future, have `TemplateOrchestrator` orchestrating other `TemplateOrchestrator` instances. Meaning that we should not consider a `TemplateOrchestrator` instance to represent the template of a full QEC.